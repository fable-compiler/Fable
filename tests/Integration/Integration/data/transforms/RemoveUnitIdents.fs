module RemoveUnitIdents

type Model = unit

let update (model: Model) =
    model, ()

update () |> ignore