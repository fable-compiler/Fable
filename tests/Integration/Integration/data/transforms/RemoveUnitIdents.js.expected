
export function update() {
    return [undefined, undefined];
}

update();

